spec/shellspec.Dockerfile

```diff
@@ -2,5 +2,5 @@
 # Don't forget to log in to the Azure Container Registry before building this image:
 # az acr login --name aksdataplanedev
 FROM aksdataplanedev.azurecr.io/shellspec/shellspec-debian:0.28.1
-RUN apt-get update && apt-get install -y --no-install-recommends gawk jq curl && apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN apt-get -o DPkg::Lock::Timeout=60 update && apt-get install -o DPkg::Lock::Timeout=60 -y --no-install-recommends gawk jq curl && apt-get -o DPkg::Lock::Timeout=60 clean && rm -rf /var/lib/apt/lists/*
```

Comment:
please add a config on the client vm instead of updating all single apt-get calls. this makes the code hard to read

https://tecadmin.net/apt-configuration-file-options/
https://manpages.ubuntu.com/manpages/trusty/man5/apt.conf.5.html